UefiPayloadPkg/SPI/kconfig.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __KCONFIG_H__
```

Comment:
No need for that file I think. apu comes with a fixed config, so we should not need any CONFIG() macros for checks

Reply:
@IgorBagnucki

Reply:
Deleted